core/model/src/main/java/org/eclipse/rdf4j/TransactionSetting.java

```diff
@@ -0,0 +1,12 @@
+package org.eclipse.rdf4j;
+
+public interface TransactionSetting {
+
+	default String getName() {
```

Comment:
@jeenbroekstra we really shouldn’t make these methods default in a brand new interface.

Reply:
Why not? If this were an abstract class it would be natural. How significantly different from these defaults are you suspecting  implementing classes to be for these methods?

Reply:
I've been thinking a bit more about this and I'm actually not so sure anymore. In general they are decent defaults, except in the case of IsolationLevel since the implementing class is IsolationLevels and not the interface IsolationLevel which blocks users from adding their own isolation levels.